sql/core/src/main/scala/org/apache/spark/sql/UDFRegistration.scala

```diff
@@ -414,6 +418,84 @@ class UDFRegistration private[sql] (functionRegistry: FunctionRegistry) extends
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
```

Comment:
It would be nice to turn style back on here since most of this function is not auto generated.